frame/nfts/src/lib.rs

```diff
@@ -0,0 +1,1725 @@
+		OptionQuery,
+	>;
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config<I>, I: 'static = ()> {
+		/// A `collection` was created.
+		Created { collection: T::CollectionId, creator: T::AccountId, owner: T::AccountId },
+		/// A `collection` was force-created.
+		ForceCreated { collection: T::CollectionId, owner: T::AccountId },
+		/// A `collection` was destroyed.
+		Destroyed { collection: T::CollectionId },
+		/// An `item` was issued.
+		Issued { collection: T::CollectionId, item: T::ItemId, owner: T::AccountId },
+		/// An `item` was transferred.
+		Transferred {
+			collection: T::CollectionId,
+			item: T::ItemId,
+			from: T::AccountId,
+			to: T::AccountId,
+		},
+		/// An `item` was destroyed.
+		Burned { collection: T::CollectionId, item: T::ItemId, owner: T::AccountId },
+		/// An `item` became non-transferable.
+		ItemTransferLocked { collection: T::CollectionId, item: T::ItemId },
+		/// An `item` became transferable.
+		ItemTransferUnlocked { collection: T::CollectionId, item: T::ItemId },
+		/// `item` metadata or attributes were locked.
+		ItemPropertiesLocked {
+			collection: T::CollectionId,
+			item: T::ItemId,
+			lock_metadata: bool,
+			lock_attributes: bool,
+		},
+		/// Some `collection` was locked.
+		CollectionLocked { collection: T::CollectionId },
+		/// The owner changed.
+		OwnerChanged { collection: T::CollectionId, new_owner: T::AccountId },
+		/// The management team changed.
+		TeamChanged {
+			collection: T::CollectionId,
+			issuer: T::AccountId,
+			admin: T::AccountId,
+			freezer: T::AccountId,
+		},
+		/// An `item` of a `collection` has been approved by the `owner` for transfer by
+		/// a `delegate`.
+		ApprovedTransfer {
+			collection: T::CollectionId,
+			item: T::ItemId,
+			owner: T::AccountId,
+			delegate: T::AccountId,
+			deadline: Option<<T as SystemConfig>::BlockNumber>,
+		},
+		/// An approval for a `delegate` account to transfer the `item` of an item
+		/// `collection` was cancelled by its `owner`.
+		ApprovalCancelled {
+			collection: T::CollectionId,
+			item: T::ItemId,
+			owner: T::AccountId,
+			delegate: T::AccountId,
+		},
+		/// All approvals of an item got cancelled.
+		AllApprovalsCancelled { collection: T::CollectionId, item: T::ItemId, owner: T::AccountId },
+		/// A `collection` has had its config changed by the `Force` origin.
+		CollectionConfigChanged { collection: T::CollectionId },
+		/// New metadata has been set for a `collection`.
+		CollectionMetadataSet { collection: T::CollectionId, data: BoundedVec<u8, T::StringLimit> },
+		/// Metadata has been cleared for a `collection`.
+		CollectionMetadataCleared { collection: T::CollectionId },
+		/// New metadata has been set for an item.
+		MetadataSet {
+			collection: T::CollectionId,
+			item: T::ItemId,
+			data: BoundedVec<u8, T::StringLimit>,
+		},
+		/// Metadata has been cleared for an item.
+		MetadataCleared { collection: T::CollectionId, item: T::ItemId },
```

Comment:
Since there is `CollectionMetadata...`, these could be `ItemMetadata...` to make it clear on interpreting these.